src/js/tabs/pid_tuning.js

```diff
@@ -1364,15 +1364,15 @@ TABS.pid_tuning.initialize = function (callback) {
             updatePidDisplay();
         });
 
-        $('#resetProfile').on('click', function(){
+        $('#resetPidProfile').on('click', function(){
             self.updating = true;
             self.sliderRetainConfiguration = true;
 
             MSP.promise(MSPCodes.MSP_SET_RESET_CURR_PID).then(function () {
                 self.refresh(function () {
                     self.updating = false;
 
-                    GUI.log(i18n.getMessage('pidTuningProfileReset'));
+                    GUI.log(i18n.getMessage('pidTuningPidProfileReset'));
```

Comment:
Is this message valid for previous firmware? Otherwise we should maintain both and check with semver.